src/tablecloth/api/join_separate.clj

```diff
@@ -61,12 +61,16 @@
        (process-group-data ds #(process-join-columns % target-column join-function col-names drop-columns?) parallel?)
        (process-join-columns ds target-column join-function col-names drop-columns?)))))
 
-;;
+(defn- prefix [prefix-name value]
```

Comment:
Still two copies of `prefix` function exists. Remove the second one below.
You can run tests with `lein midje` to verify if everything works.